_posts/Durr_Hoyer_Post_1.md

```diff
@@ -0,0 +1,126 @@
+---
+title: "A quantum algorithm for finding the minimum of an unsorted table"
+date:
```

Comment:
We will bump this on the day it goes live, but it breaks the test builds if it is not there
```suggestion
date: 2020-06-22
```